### ISSUE TITLE
update-vals etc from Clojure 1.11 missing

### ISSUE THREAD

**Issue by Oddsor** [borkdude/jet#137]
Tried writing a little transformer that was going to use `update-vals`, but it seems to be missing from Jet!

Similar story to other functions added in Clojure 1.11 such as `update-keys`, `parse-long` etc...

**Comment by borkdude** [comment]
Can you make a repro?

**Comment by Oddsor** [comment]
I made a PR with tests instead!
https://github.com/borkdude/jet/pull/138

**Comment by borkdude** [comment]
Thanks :)

I noticed the changelogs weren't updated yet. Can you do this too?

**Comment by Oddsor** [comment]
Will do!
Also noted in the PR that maybe a separate NS should be created instead of inlining the missing functions directly?
Babashka does this with a `core-extras` in an ["impl"-namespace](https://github.com/babashka/babashka/blob/master/src/babashka/impl/clojure/core.clj#L138C13-L138C13)

**Comment by Oddsor** [comment]
Okay, so maybe this PR should be closed and then I can open one for SCI where the functions are added to `clojure-core` here?
https://github.com/babashka/sci/blob/master/src/sci/impl/namespaces.cljc#L991C11-L991C11

**Comment by borkdude** [comment]
No, not in SCI, but in https://github.com/babashka/sci.configs.

**Comment by Oddsor** [comment]
Cool, I'll give updating sci.configs a try later today 😊

**Comment by Oddsor** [comment]
I tried to applying the recent update to [sci.configs](https://github.com/babashka/sci.configs/pull/20) to Jet and noticed that leiningen is still used to build and test, which doesn't support referencing git-dependencies!

I can try to assist with moving to building using deps.edn instead if that's a good direction for the project, but for now it looks like this issue will be resolved with the next release 😊

**Comment by borkdude** [comment]
Ah yes, sorry, I didn't notice that. We can move to tools.build. I'll make an issue for that and solve that myself, if nobody else jumps on it.
